CHANGELOG.md

```diff
@@ -6,64 +6,78 @@
 * Fixed issue with Clustergram rows and columns reordering incorrectly
   on the heatmap when precomputed traces are used to generate the
   figure.
+* Fixed issue with Speck not rendering unless it is attached to a callback.
+* Fixed issue with Speck trying to calculate a system with no atoms,
+  which led to an error.
```

Comment:
I suggest removing this line.

Reply:
@mkcor This is an issue that was fixed, though -- do you have an idea of how it could be worded better? (Or does this belong in another section?)

Reply:
My rewording suggestion corresponds to making the whole sentence fit in one line, which is "Prevent Speck from trying to calculate a system with no atom." It says it all. We fixed an issue (when there was an issue, there was an error, but it's trivial so there's no need to specify it; it's not surprising that trying to calculate an empty system would lead to an error; it's not interesting to write out "there used to be an error" and the source code is clear and commented...).

"Prevent Speck from trying to calculate a system with no atom" makes it clear that *previously* Speck was trying to calculate such a system and clearly that was an issue!!

Reply:
Thanks for the clarification :) Fixed in 8a1f343!